README-DEV.md

```diff
@@ -24,7 +24,7 @@ NOTE: Integration tests are currently only partially supported.
 Notes about integration tests
 * Config needs to be set with variable `IT_SNOWFLAKE_CONF` defined and
     pointing to the location of the Snowflake configuration file.
-* An example configuration file is provided in `snowflake.conf.example`.
+* An example configuration file is provided in [snowflake.conf.example]().
```

Comment:
`snowflake.conf.example` is not used any more.
Please remove this line, and remove the added example file.